services/libs/tinybird/pipes/categories_oss_index.pipe

```diff
@@ -0,0 +1,88 @@
+NODE categories_oss_index_agregates
+SQL >
+    SELECT categoryId, coalesce(sum(contributorCount), 0) as "totalContributors"
+    FROM segments_aggregates_with_ids_datasource
+    WHERE categoryId != ''
+    GROUP BY categoryId
+    ORDER BY totalContributors DESC
+
+NODE categories_oss_index_projects_data
```

Comment:
better to call this node `insightsProjects_deduplicated` since it only has FINAL and selects some fields